djangocms_url_manager/models.py

```diff
@@ -88,8 +88,16 @@ class Meta:
         abstract = True
 
 
-class Url(AbstractUrl):
+class UrlGrouper(AbstractUrl):
+    pass
+
 
+class Url(AbstractUrl):
+    url_grouper = models.ForeignKey(
+        UrlGrouper,
+        on_delete=models.CASCADE,
+        related_name='%(app_label)s_%(class)s_related',
```

Comment:
The related name should also include something unique to this current model. Imagine if you had multiple foreignkeys, they would all result in: "djangocms_url_manager_urls_related", this will clash if you have more than one. To make them unique we can use the field name too: "djangocms_url_manager_urls_url_grouper".


```suggestion
        related_name='%(app_label)s_%(class)s_url_grouper,
```